src/core/load-fragment.c

```diff
@@ -1211,6 +1212,35 @@ int config_parse_exec_cpu_sched_policy(const char *unit,
         return 0;
 }
 
+int config_parse_numa_mask(const char *unit,
+                           const char *filename,
+                           unsigned line,
+                           const char *section,
+                           unsigned section_line,
+                           const char *lvalue,
+                           int ltype,
+                           const char *rvalue,
+                           void *data,
+                           void *userdata) {
+        int r;
+        NUMAPolicy *p = data;
+
+        assert(filename);
+        assert(lvalue);
+        assert(rvalue);
+        assert(data);
+
+        r = parse_cpu_set_extend(rvalue, &p->nodes, true, unit, filename, line, lvalue);
+        if (r < 0) {
+                log_syntax(unit, LOG_ERR, filename, line, 0, "Failed to parse NUMA node mask, ignoring: %s", rvalue);
```

Comment:
why pass the error here as `0`? pass `r` here, there's no reason not to pass it to log_syntax(). (Note that log_syntax will include the error code in a structured logging field ERRNO= if it has it, even if `%m` is not used in the message format string)